articles/dashboard/guides/applications/enable-android-app-links.md

```diff
@@ -37,7 +37,7 @@ keytool -list -v -keystore my-release-key.keystore
 For more info about signing certificates, see Android's [Sign Your App](https://developer.android.com/studio/publish/app-signing.html) developer doc.
 :::
 
-![Add Mobile Settings](/media/articles/applications/mobile-settings.png)
+![Add Device Settings](/media/articles/applications/mobile-settings.png)
```

Comment:
This image referenced here is still showing the old name. Please update the resource following the steps [here](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#screenshots).

Reply:
Sorry, I should have left the PR in draft mode until I had a chance to get to that!

Images have been added.